enclave_wrangler/datasets.py

```diff
@@ -37,15 +33,15 @@
 #     #'content-type': 'application/json'
 # }
 DEBUG = False
-# TODO: Once git LFS set up, dl directly to datasets folder, or put these in raw/ and move csvs_repaired to datasets/
 CSV_DOWNLOAD_DIR = os.path.join(TERMHUB_CSETS_DIR, 'datasets', 'downloads')
 CSV_TRANSFORM_DIR = os.path.join(TERMHUB_CSETS_DIR, 'datasets', 'prepped_files')
 DESC = 'Tool for working w/ the Palantir Foundry enclave API. This part is for downloading enclave datasets.'
-os.makedirs(CSV_TRANSFORM_DIR, exist_ok=True)
+for d in [CSV_DOWNLOAD_DIR, CSV_TRANSFORM_DIR]:
+    os.makedirs(d, exist_ok=True)
```

Comment:
This was a bug I had to fix. The download dir didn't exist, so needed to be created. Kind of confusing, because I went to look back at the GH action log from when I first ran this, and it somehow ran successfully several times before.